airflow/cli/commands/task_command.py

```diff
@@ -174,6 +175,14 @@ def _run_raw_task(args, ti: TaskInstance) -> None:
     )
 
 
+def _extract_external_executor_id(args) -> Optional[str]:
+    if "external_executor_id" in args:
+        return args.external_executor_id
+    elif "external_executor_id" in os.environ:
+        return args.external_executor_id
```

Comment:
Should we read env variable here?

Reply:
Good catch! Changed

Reply:
What did you think of the overal implementation @mik-laj ?

Reply:
Fixed the reference.